src/zui/ZUIMenu/index.tsx

```diff
@@ -84,12 +73,7 @@ const ZUIMenu: FC<ZUIMenuProps> = ({
           }}
         >
           {item.startIcon && <ListItemIcon>{item.startIcon}</ListItemIcon>}
```

Comment:
The icons on the left-hand side should have 24px (1.5rem) in font-size. Right now I'm reading out these values:
<img width="266" alt="image" src="https://github.com/user-attachments/assets/84a3bd12-7612-4892-87c5-e219682d1355">